app/(afterLogin)/(deal)/deals/manage-deals/_components/sideBar/index.tsx

```
@@ -0,0 +1,200 @@
+        {/* Solution & Supplier */}
+        <div className="grid grid-cols-2 gap-4">
+          <Form.Item name="solution" label="Solution">
+            <Select placeholder="Solution" />
+          </Form.Item>
+          <Form.Item name="supplier" label="Supplier">
+            <Select placeholder="Supplier" />
+          </Form.Item>
+        </div>
+
+        {/* Roles */}
+          <div className="flex justify-between items-center ">
```

ESLint

[error] Delete `··`

(prettier/prettier)